From 80467269b1d75b01c67720b028912acdcadda9a5 Mon Sep 17 00:00:00 2001
From: Tejas Patel <tejas.patel1@einfochips.com>
Date: Wed, 10 Nov 2021 15:00:43 +0530
Subject: [PATCH 04/16] IMX8 THOR96: Enable EEPROM AT24 Support

Command for testing :
For Write
$ echo hello > /sys/bus/i2c/devices/i2c-1/1-0050/eeprom

For read
$ cat /sys/bus/i2c/devices/i2c-1/1-0050/eeprom | hexdump -C

Change-Id: Idece853e75ee2cd8c3fb11d6e43221201623a51f
Signed-off-by: Tejas Patel <tejas.patel1@einfochips.com>
Signed-off-by: Tanvi Chauhan <tanvi.chauhan@einfochips.com>

diff --git a/arch/arm64/boot/dts/freescale/imx8mq-thor96.dts b/arch/arm64/boot/dts/freescale/imx8mq-thor96.dts
index 51622b6b0..cd3f75a88 100644
--- a/arch/arm64/boot/dts/freescale/imx8mq-thor96.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mq-thor96.dts
@@ -240,6 +240,8 @@
 	eeprom: eeprom@50 {
 		compatible = "atmel,24c256";
 		reg = <0x50>;
+		pagesize = <32>;
+		status = "okay";
 	};
 };
 
-- 
2.17.1

